templates/rhsm-katello-reconfigure.erb

```diff
@@ -31,6 +31,17 @@ then
 
   # configure rhsm
   # the config command was introduced in rhsm 0.96.6
+  subscription-manager config \
```

Comment:
can we DRY this? Maybe wrap just `BASEURL= foo` around an if else?

Reply:
sure